plugins/mcreator-localization/lang/texts.properties

```diff
@@ -2312,7 +2312,9 @@ elementgui.living_entity.immune_trident=Trident
 elementgui.living_entity.immune_anvil=Anvil
 elementgui.living_entity.immune_wither=Wither
 elementgui.living_entity.immune_dragon_breath=Dragon Breath
-elementgui.living_entity.is_water_mob=Is water entity (breathes underwater, is not pushed by water flow)
+elementgui.living_entity.is_water_mob=Is water entity
+elementgui.living_entity.condition_can_breathe_underwater=Breathe underwater condition
```

Comment:
As this is checkbox-style input, I would omit word condition and instead call it:

Does breathe underwater

and

Is pushed by fluids